models/staging/stg_workday__personal_information.sql

```diff
@@ -36,7 +36,7 @@ final as (
         date_of_birth,
         date_of_death,
         gender,
-        hispanic_or_latino,
+        hispanic_or_latino, 
```

Comment:
This is a boolean field and should be renamed to be `is_hispanic_or_latino`. Can you go through the fields and be sure to rename any of the boolean fields to have this same naming convention for consistency.

Reply:
Updated

Reply:
@fivetran-joemarkiewicz I wonder if we can automate this to automatically populate (if macro field is boolean, append with the renaming in the staging model)

Reply:
Yes! This is something that should be easily automated for future packages. Will add as a feature to the automation repo